diff --git a/node_modules/@bangle.dev/tooltip/dist/index.js b/node_modules/@bangle.dev/tooltip/dist/index.js
index 6a5dbce..76ca940 100644
--- a/node_modules/@bangle.dev/tooltip/dist/index.js
+++ b/node_modules/@bangle.dev/tooltip/dist/index.js
@@ -274,7 +274,7 @@ function getSelectionReferenceElement(view) {
     return {
         getBoundingClientRect: () => {
             const { selection } = view.state;
-            let { head, from } = selection;
+            let { head, from, to } = selection;
             // since head is dependent on the users choice of direction,
             // it is not always equal to `from`.
             // For textSelections we want to show the tooltip at head of the
@@ -282,7 +282,9 @@ function getSelectionReferenceElement(view) {
             // But for NodeSelection we always want `from` since, if we go with `head`
             // coordsAtPos(head) might get the position `to` in head, resulting in
             // incorrectly getting position of the node after the selected Node.
-            const pos = selection instanceof NodeSelection ? from : head;
+
+            // Using head actually puts the floating menu wherever the cursor ends - use from for now: https://prosemirror.net/docs/ref/version/0.20.0.html
+            const pos = from;
             const start = view.coordsAtPos(pos);
             let { top, bottom, left, right } = start;
             let width = right - left;
