diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/README.md b/node_modules/@ethereum-attestation-service/eas-sdk/README.md
index adbe142..0c33398 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/README.md
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/README.md
@@ -32,7 +32,7 @@ Import and initialize the library
 
 ```javascript
 import { EAS, Offchain, SchemaEncoder, SchemaRegistry } from "@ethereum-attestation-service/eas-sdk";
-import { ethers } from 'ethers';
+import { ethers } from 'ethers-v6';
 
 export const EASContractAddress = "0xC2679fBD37d54388Ce493F1DB75320D236e1815e"; // Sepolia v0.26
 
@@ -334,7 +334,7 @@ Here's an example of how to register a new schema:
 
 ```javascript
 import { SchemaRegistry } from "@ethereum-attestation-service/eas-sdk";
-import { ethers } from 'ethers';
+import { ethers } from 'ethers-v6';
 
 const schemaRegistryContractAddress = "0xYourSchemaRegistryContractAddress";
 const schemaRegistry = new SchemaRegistry(schemaRegistryContractAddress);
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/eas.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/eas.d.ts
index 02794f1..5336522 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/eas.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/eas.d.ts
@@ -1,10 +1,10 @@
 import { EAS as EASContract } from '@ethereum-attestation-service/eas-contracts';
-import { Overrides } from 'ethers';
+import { Overrides } from 'ethers-v6';
 import { EIP712Proxy } from './eip712-proxy';
 import { Delegated, Offchain } from './offchain';
 import { AttestationRequest, DelegatedAttestationRequest, DelegatedProxyAttestationRequest, DelegatedProxyRevocationRequest, DelegatedRevocationRequest, MultiAttestationRequest, MultiDelegatedAttestationRequest, MultiDelegatedProxyAttestationRequest, MultiDelegatedProxyRevocationRequest, MultiDelegatedRevocationRequest, MultiRevocationRequest, RevocationRequest } from './request';
 import { Base, SignerOrProvider, Transaction } from './transaction';
-export { Overrides } from 'ethers';
+export { Overrides } from 'ethers-v6';
 export * from './request';
 export interface Attestation {
     uid: string;
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/eip712-proxy.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/eip712-proxy.d.ts
index aaa3ba4..300dfe9 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/eip712-proxy.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/eip712-proxy.d.ts
@@ -1,5 +1,5 @@
 import { EIP712Proxy as EIP712ProxyContract } from '@ethereum-attestation-service/eas-contracts';
-import { Overrides } from 'ethers';
+import { Overrides } from 'ethers-v6';
 import { DelegatedProxy } from './offchain';
 import { DelegatedProxyAttestationRequest, DelegatedProxyRevocationRequest, MultiDelegatedProxyAttestationRequest, MultiDelegatedProxyRevocationRequest } from './request';
 import { Base, SignerOrProvider, Transaction } from './transaction';
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/indexer.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/indexer.d.ts
index 0491faf..1cfef77 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/indexer.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/indexer.d.ts
@@ -1,5 +1,5 @@
 import { Indexer as IndexerContract } from '@ethereum-attestation-service/eas-contracts';
-import { Overrides } from 'ethers';
+import { Overrides } from 'ethers-v6';
 import { Base, SignerOrProvider, Transaction } from './transaction';
 export interface IndexerOptions {
     signerOrProvider?: SignerOrProvider;
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/legacy/version.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/legacy/version.d.ts
index b4c70e2..876a0da 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/legacy/version.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/legacy/version.d.ts
@@ -1,2 +1,2 @@
-import { BaseContract } from 'ethers';
+import { BaseContract } from 'ethers-v6';
 export declare const legacyVersion: (contract: BaseContract) => Promise<string | undefined>;
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/delegated-proxy.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/delegated-proxy.d.ts
index c4e6703..cb483db 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/delegated-proxy.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/delegated-proxy.d.ts
@@ -1,4 +1,4 @@
-import { Signer } from 'ethers';
+import { Signer } from 'ethers-v6';
 import { EIP712AttestationParams, EIP712RevocationParams } from './delegated';
 import { EIP712MessageTypes, EIP712Response, TypedDataConfig, TypedDataHandler } from './typed-data-handler';
 export { EIP712MessageTypes, EIP712TypedData, EIP712Request, EIP712Response, TypedDataConfig } from './typed-data-handler';
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/delegated.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/delegated.d.ts
index c1970c3..ee9b9f1 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/delegated.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/delegated.d.ts
@@ -1,4 +1,4 @@
-import { Signer } from 'ethers';
+import { Signer } from 'ethers-v6';
 import { EIP712MessageTypes, EIP712Params, EIP712Response, PartialTypedDataConfig, TypedDataHandler } from './typed-data-handler';
 export { EIP712MessageTypes, EIP712TypedData, EIP712Request, EIP712Response, PartialTypedDataConfig } from './typed-data-handler';
 export declare const EIP712_NAME = "EAS";
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/offchain.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/offchain.d.ts
index ebaafdf..8d8a5b7 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/offchain.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/offchain.d.ts
@@ -1,4 +1,4 @@
-import { Signer } from 'ethers';
+import { Signer } from 'ethers-v6';
 import { EAS } from '../eas';
 import { DomainTypedData, EIP712MessageTypes, EIP712Params, EIP712Response, EIP712Types, PartialTypedDataConfig, TypedDataHandler } from './typed-data-handler';
 export { EIP712Request, PartialTypedDataConfig, EIP712MessageTypes } from './typed-data-handler';
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/typed-data-handler.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/typed-data-handler.d.ts
index 45fc037..dfed495 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/typed-data-handler.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/offchain/typed-data-handler.d.ts
@@ -1,4 +1,4 @@
-import { Signer } from 'ethers';
+import { Signer } from 'ethers-v6';
 export interface PartialTypedDataConfig {
     address: string;
     version: string;
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/schema-registry.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/schema-registry.d.ts
index 6be11b2..f87cdcc 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/schema-registry.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/schema-registry.d.ts
@@ -1,5 +1,5 @@
 import { SchemaRegistry as SchemaRegistryContract } from '@ethereum-attestation-service/eas-contracts';
-import { Overrides } from 'ethers';
+import { Overrides } from 'ethers-v6';
 import { Base, SignerOrProvider, Transaction } from './transaction';
 export declare type SchemaRecord = {
     uid: string;
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/transaction.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/transaction.d.ts
index de13254..d9a779e 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/transaction.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/transaction.d.ts
@@ -1,4 +1,4 @@
-import { BaseContract, ContractFactory, Provider, Signer, TransactionReceipt, TransactionResponse } from 'ethers';
+import { BaseContract, ContractFactory, Provider, Signer, TransactionReceipt, TransactionResponse } from 'ethers-v6';
 export declare type SignerOrProvider = Signer | Provider;
 export declare class Transaction<T> {
     readonly tx: TransactionResponse;
diff --git a/node_modules/@ethereum-attestation-service/eas-sdk/dist/utils.d.ts b/node_modules/@ethereum-attestation-service/eas-sdk/dist/utils.d.ts
index dec0b4b..4bdcc5f 100644
--- a/node_modules/@ethereum-attestation-service/eas-sdk/dist/utils.d.ts
+++ b/node_modules/@ethereum-attestation-service/eas-sdk/dist/utils.d.ts
@@ -1,4 +1,4 @@
-import { TransactionReceipt, TransactionResponse } from 'ethers';
+import { TransactionReceipt, TransactionResponse } from 'ethers-v6';
 export declare const ZERO_ADDRESS: string;
 export declare const ZERO_BYTES = "0x";
 export declare const ZERO_BYTES32 = "0x0000000000000000000000000000000000000000000000000000000000000000";
