diff --git a/node_modules/@lens-protocol/client/dist/declarations/src/gated/LensClient.d.ts b/node_modules/@lens-protocol/client/dist/declarations/src/gated/LensClient.d.ts
index 6bb05af..7b787fd 100644
--- a/node_modules/@lens-protocol/client/dist/declarations/src/gated/LensClient.d.ts
+++ b/node_modules/@lens-protocol/client/dist/declarations/src/gated/LensClient.d.ts
@@ -25,7 +25,7 @@ export type LensClientConfig = base.LensClientConfig & {
  * @example
  * NodeJS example:
  * ```ts
- * import { Wallet } from 'ethers';
+ * import { Wallet } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
@@ -46,7 +46,7 @@ export type LensClientConfig = base.LensClientConfig & {
  * @example
  * Browser example with `ethers` v6 and browser wallet (e.g. MetaMask):
  * ```ts
- * import { BrowserProvider } from 'ethers';
+ * import { BrowserProvider } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
diff --git a/node_modules/@lens-protocol/client/dist/declarations/src/gated/index.d.ts b/node_modules/@lens-protocol/client/dist/declarations/src/gated/index.d.ts
index 2b99542..40818f8 100644
--- a/node_modules/@lens-protocol/client/dist/declarations/src/gated/index.d.ts
+++ b/node_modules/@lens-protocol/client/dist/declarations/src/gated/index.d.ts
@@ -28,7 +28,7 @@
  *
  * Typical NodeJS setup:
  * ```ts
- * import { Wallet } from 'ethers';
+ * import { Wallet } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
@@ -48,7 +48,7 @@
  *
  * Browser setup with `ethers` v6 and an EIP-1193 wallet (e.g. MetaMask):
  * ```ts
- * import { BrowserProvider } from 'ethers';
+ * import { BrowserProvider } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
diff --git a/node_modules/@lens-protocol/client/dist/declarations/src/types.d.ts b/node_modules/@lens-protocol/client/dist/declarations/src/types.d.ts
index 086f73e..fa3a256 100644
--- a/node_modules/@lens-protocol/client/dist/declarations/src/types.d.ts
+++ b/node_modules/@lens-protocol/client/dist/declarations/src/types.d.ts
@@ -1,4 +1,4 @@
-import type { TypedDataDomain, TypedDataField } from 'ethers';
+import type { TypedDataDomain, TypedDataField } from 'ethers-v5';
 export type TypedData = {
     domain: TypedDataDomain;
     types: Record<string, Array<TypedDataField>>;
diff --git a/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.cjs.dev.js b/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.cjs.dev.js
index fd3fd97..5178534 100644
--- a/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.cjs.dev.js
+++ b/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.cjs.dev.js
@@ -208,7 +208,7 @@ var Gated = /*#__PURE__*/function () {
  * @example
  * NodeJS example:
  * ```ts
- * import { Wallet } from 'ethers';
+ * import { Wallet } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
@@ -229,7 +229,7 @@ var Gated = /*#__PURE__*/function () {
  * @example
  * Browser example with `ethers` v6 and browser wallet (e.g. MetaMask):
  * ```ts
- * import { BrowserProvider } from 'ethers';
+ * import { BrowserProvider } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
diff --git a/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.cjs.prod.js b/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.cjs.prod.js
index bd2554a..7b7e81e 100644
--- a/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.cjs.prod.js
+++ b/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.cjs.prod.js
@@ -208,7 +208,7 @@ var Gated = /*#__PURE__*/function () {
  * @example
  * NodeJS example:
  * ```ts
- * import { Wallet } from 'ethers';
+ * import { Wallet } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
@@ -229,7 +229,7 @@ var Gated = /*#__PURE__*/function () {
  * @example
  * Browser example with `ethers` v6 and browser wallet (e.g. MetaMask):
  * ```ts
- * import { BrowserProvider } from 'ethers';
+ * import { BrowserProvider } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
diff --git a/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.esm.js b/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.esm.js
index 635899b..4c3a755 100644
--- a/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.esm.js
+++ b/node_modules/@lens-protocol/client/gated/dist/lens-protocol-client-gated.esm.js
@@ -206,7 +206,7 @@ var Gated = /*#__PURE__*/function () {
  * @example
  * NodeJS example:
  * ```ts
- * import { Wallet } from 'ethers';
+ * import { Wallet } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
@@ -227,7 +227,7 @@ var Gated = /*#__PURE__*/function () {
  * @example
  * Browser example with `ethers` v6 and browser wallet (e.g. MetaMask):
  * ```ts
- * import { BrowserProvider } from 'ethers';
+ * import { BrowserProvider } from 'ethers-v5';
  * import { production } from '@lens-protocol/client';
  * import { LensClient } from '@lens-protocol/client/gated';
  *
