diff --git a/node_modules/@lens-protocol/react-web/dist/declarations/src/index.d.ts b/node_modules/@lens-protocol/react-web/dist/declarations/src/index.d.ts
index 4a020f2..d91c1b6 100644
--- a/node_modules/@lens-protocol/react-web/dist/declarations/src/index.d.ts
+++ b/node_modules/@lens-protocol/react-web/dist/declarations/src/index.d.ts
@@ -54,7 +54,7 @@
  * @example
  * An example of how to create custom bindings for a `Wallet` from `ethers.js`
  * ```ts
- * import { InfuraProvider, Wallet } from 'ethers';
+ * import { InfuraProvider, Wallet } from 'ethers-v5';
  * import { IBindings } from '@lens-protocol/react-web';
  *
  * const provider = new providers.InfuraProvider('maticmum');
