diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/TypedData.d.ts b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/TypedData.d.ts
index 4e54d90..bb7b364 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/TypedData.d.ts
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/TypedData.d.ts
@@ -1,4 +1,4 @@
-import { TypedDataField } from 'ethers';
+import { TypedDataField } from 'ethers-v5';
 export type Eip712TypedDataDomain = {
     name: string;
     version: string;
diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/Erc20.d.ts b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/Erc20.d.ts
index 353c1ac..5c2f40c 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/Erc20.d.ts
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/Erc20.d.ts
@@ -1,4 +1,4 @@
-import type { BaseContract, BigNumber, BigNumberish, BytesLike, CallOverrides, ContractTransaction, Overrides, PopulatedTransaction, Signer, utils } from 'ethers';
+import type { BaseContract, BigNumber, BigNumberish, BytesLike, CallOverrides, ContractTransaction, Overrides, PopulatedTransaction, Signer, utils } from 'ethers-v5';
 import type { FunctionFragment, Result, EventFragment } from '@ethersproject/abi';
 import type { Listener, Provider } from '@ethersproject/providers';
 import type { TypedEventFilter, TypedEvent, TypedListener, OnEvent, PromiseOrValue } from './common';
diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensFollowNFT.d.ts b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensFollowNFT.d.ts
index a960697..0ab7c7b 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensFollowNFT.d.ts
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensFollowNFT.d.ts
@@ -1,4 +1,4 @@
-import type { BaseContract, BigNumber, BigNumberish, BytesLike, CallOverrides, ContractTransaction, Overrides, PopulatedTransaction, Signer, utils } from 'ethers';
+import type { BaseContract, BigNumber, BigNumberish, BytesLike, CallOverrides, ContractTransaction, Overrides, PopulatedTransaction, Signer, utils } from 'ethers-v5';
 import type { FunctionFragment, Result, EventFragment } from '@ethersproject/abi';
 import type { Listener, Provider } from '@ethersproject/providers';
 import type { TypedEventFilter, TypedEvent, TypedListener, OnEvent, PromiseOrValue } from './common';
diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensHub.d.ts b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensHub.d.ts
index f3c7630..dbcb3fa 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensHub.d.ts
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensHub.d.ts
@@ -1,4 +1,4 @@
-import type { BaseContract, BigNumber, BigNumberish, BytesLike, CallOverrides, ContractTransaction, Overrides, PopulatedTransaction, Signer, utils } from 'ethers';
+import type { BaseContract, BigNumber, BigNumberish, BytesLike, CallOverrides, ContractTransaction, Overrides, PopulatedTransaction, Signer, utils } from 'ethers-v5';
 import type { FunctionFragment, Result, EventFragment } from '@ethersproject/abi';
 import type { Listener, Provider } from '@ethersproject/providers';
 import type { TypedEventFilter, TypedEvent, TypedListener, OnEvent, PromiseOrValue } from './common';
diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensTokenHandleRegistry.d.ts b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensTokenHandleRegistry.d.ts
index e622e6b..db08e32 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensTokenHandleRegistry.d.ts
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/LensTokenHandleRegistry.d.ts
@@ -1,4 +1,4 @@
-import type { BaseContract, BigNumber, BigNumberish, BytesLike, CallOverrides, ContractTransaction, Overrides, PopulatedTransaction, Signer, utils } from 'ethers';
+import type { BaseContract, BigNumber, BigNumberish, BytesLike, CallOverrides, ContractTransaction, Overrides, PopulatedTransaction, Signer, utils } from 'ethers-v5';
 import type { FunctionFragment, Result } from '@ethersproject/abi';
 import type { Listener, Provider } from '@ethersproject/providers';
 import type { TypedEventFilter, TypedEvent, TypedListener, OnEvent, PromiseOrValue } from './common';
diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/common.d.ts b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/common.d.ts
index bcfb66b..80f928e 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/common.d.ts
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/declarations/src/types/common.d.ts
@@ -1,5 +1,5 @@
 import type { Listener } from '@ethersproject/providers';
-import type { Event, EventFilter } from 'ethers';
+import type { Event, EventFilter } from 'ethers-v5';
 export interface TypedEvent<TArgsArray extends Array<any> = any, TArgsObject = any> extends Event {
     args: TArgsArray & TArgsObject;
 }
diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.cjs.dev.js b/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.cjs.dev.js
index df6b1b7..d027d19 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.cjs.dev.js
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.cjs.dev.js
@@ -2,7 +2,7 @@
 
 Object.defineProperty(exports, '__esModule', { value: true });
 
-var ethers = require('ethers');
+var ethers = require('ethers-v5');
 
 var lensFollowNftAbi = [
 	{
diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.cjs.prod.js b/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.cjs.prod.js
index df6b1b7..d027d19 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.cjs.prod.js
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.cjs.prod.js
@@ -2,7 +2,7 @@
 
 Object.defineProperty(exports, '__esModule', { value: true });
 
-var ethers = require('ethers');
+var ethers = require('ethers-v5');
 
 var lensFollowNftAbi = [
 	{
diff --git a/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.esm.js b/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.esm.js
index c757229..e708b9d 100644
--- a/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.esm.js
+++ b/node_modules/@lens-protocol/blockchain-bindings/dist/lens-protocol-blockchain-bindings.esm.js
@@ -1,4 +1,4 @@
-import { Contract, utils } from 'ethers';
+import { Contract, utils } from 'ethers-v5';
 
 var lensFollowNftAbi = [
 	{
