diff --git a/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.cjs.dev.js b/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.cjs.dev.js
index 3588271..dd1ee5d 100644
--- a/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.cjs.dev.js
+++ b/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.cjs.dev.js
@@ -3,7 +3,7 @@
 Object.defineProperty(exports, '__esModule', { value: true });
 
 var sharedKernel = require('@lens-protocol/shared-kernel');
-var ethers = require('ethers');
+var ethers = require('ethers-v5');
 var wagmi = require('wagmi');
 var actions = require('wagmi/actions');
 
diff --git a/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.cjs.prod.js b/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.cjs.prod.js
index 3588271..dd1ee5d 100644
--- a/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.cjs.prod.js
+++ b/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.cjs.prod.js
@@ -3,7 +3,7 @@
 Object.defineProperty(exports, '__esModule', { value: true });
 
 var sharedKernel = require('@lens-protocol/shared-kernel');
-var ethers = require('ethers');
+var ethers = require('ethers-v5');
 var wagmi = require('wagmi');
 var actions = require('wagmi/actions');
 
diff --git a/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.esm.js b/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.esm.js
index 183a3a2..539b74a 100644
--- a/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.esm.js
+++ b/node_modules/@lens-protocol/wagmi/dist/lens-protocol-wagmi.esm.js
@@ -1,5 +1,5 @@
 import { invariant } from '@lens-protocol/shared-kernel';
-import { providers } from 'ethers';
+import { providers } from 'ethers-v5';
 import { SwitchChainNotSupportedError } from 'wagmi';
 import { getPublicClient, getWalletClient, getNetwork, switchNetwork } from 'wagmi/actions';
 
