diff --git a/node_modules/@lit-protocol/access-control-conditions/src/lib/humanizer.js b/node_modules/@lit-protocol/access-control-conditions/src/lib/humanizer.js
index 9af696b..a558a72 100644
--- a/node_modules/@lit-protocol/access-control-conditions/src/lib/humanizer.js
+++ b/node_modules/@lit-protocol/access-control-conditions/src/lib/humanizer.js
@@ -4,6 +4,7 @@ exports.humanizeAccessControlConditions = exports.humanizeUnifiedAccessControlCo
 const constants_1 = require("@lit-protocol/constants");
 const misc_1 = require("@lit-protocol/misc");
 const utils_1 = require("ethers/lib/utils");
+
 /**
  *
  * Format SOL number using Ether Units
@@ -189,7 +190,7 @@ const humanizeEvmBasicAccessControlConditions = async ({ accessControlConditions
         }
         else if (acc.standardContractType === '' &&
             acc.method === 'eth_getBalance') {
-            return `Owns ${(0, exports.humanizeComparator)(acc.returnValueTest.comparator)} ${(0, utils_1.formatEther)(acc.returnValueTest.value)} ETH`;
+            return `Owns ${(0, exports.humanizeComparator)(acc.returnValueTest.comparator)} ${(0, utils_1.formatEther)(acc.returnValueTest.value)} ${constants_1.LIT_CHAINS[acc.chain].symbol}`;
         }
         else if (acc.standardContractType === '' && acc.method === '') {
             if (myWalletAddress &&
