diff --git a/node_modules/lit-share-modal-v3/dist/lit-share-modal-v3.es.js b/node_modules/lit-share-modal-v3/dist/lit-share-modal-v3.es.js
index def24d1..e3cce00 100644
--- a/node_modules/lit-share-modal-v3/dist/lit-share-modal-v3.es.js
+++ b/node_modules/lit-share-modal-v3/dist/lit-share-modal-v3.es.js
@@ -13439,14 +13439,6 @@ const chainConfig = {
     types: ethereumTypesConfig,
     addressValidator: (walletAddress) => true
   },
-  optimism: {
-    value: "optimism",
-    label: "Optimism",
-    logo: optimismLogo,
-    abbreviation: "op",
-    types: ethereumTypesConfig,
-    addressValidator: (walletAddress) => true
-  },
   celo: {
     value: "celo",
     label: "Celo",
