diff --git a/node_modules/prosemirror-utils/dist/index.js b/node_modules/prosemirror-utils/dist/index.js
index 915a989..8b3c797 100644
--- a/node_modules/prosemirror-utils/dist/index.js
+++ b/node_modules/prosemirror-utils/dist/index.js
@@ -195,10 +195,9 @@ var canInsert = ($pos, content) => {
   const index = $pos.index();
   if (content instanceof Fragment2) {
     return $pos.parent.canReplace(index, index, content);
-  } else if (content instanceof PMNode2) {
-    return $pos.parent.canReplaceWith(index, index, content.type);
   }
-  return false;
+  // For reference, see https://github.com/charmverse/app.charmverse.io/pull/3055
+  return $pos.parent.canReplaceWith(index, index, content.type);
 };
 var isEmptyParagraph = (node) => {
   return !node || node.type.name === "paragraph" && node.nodeSize === 2;
