diff --git a/node_modules/prosemirror-utils/dist/index.js b/node_modules/prosemirror-utils/dist/index.js
index 5edc8de..35d8e7d 100644
--- a/node_modules/prosemirror-utils/dist/index.js
+++ b/node_modules/prosemirror-utils/dist/index.js
@@ -324,10 +324,8 @@ var canInsert = function canInsert($pos, content) {
 
   if (content instanceof prosemirrorModel.Fragment) {
     return $pos.parent.canReplace(index, index, content);
-  } else if (content instanceof prosemirrorModel.Node) {
-    return $pos.parent.canReplaceWith(index, index, content.type);
   }
-  return false;
+  return $pos.parent.canReplaceWith(index, index, content.type);
 };
 
 // (node: ProseMirrorNode) → boolean
